src/main/java/io/split/android/client/storage/attributes/PersistentAttributesStorage.java

```diff
@@ -0,0 +1,16 @@
+package io.split.android.client.storage.attributes;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Map;
+
+public interface PersistentAttributesStorage {
+
+    void set(@Nullable Map<String, Object> attributes);
+
+    @NonNull
+    Map<String, Object> get();
```

Comment:
Please rename this function to getAll().
```suggestion
    Map<String, Object> getAll();
```